serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc

```diff
@@ -1 +1,368 @@
-= CNCF Serverless Workflow specification support
\ No newline at end of file
+CNCF Serverless Workflow Specification Support
+==============================================
+v1.0, 06-08-2022
+:compat-mode!:
+// Metadata:
+:description: CNCF Serverless Workflow Specification Support
+:keywords: kogito, workflow, serverless, cncf, specification
+// links
+:quarkus_config_guide_url: https://quarkus.io/guides/config-reference
+
+
+In this guide, you can learn the details about the Kogito implementation of the {spec_website_url}[CNCF Serverless Workflow] specification.
+
+== Current Status
+
+Currently, Kogito implements version {spec_doc_url}[{spec_version}] of the specification. The table below classifies the current implementation status for each specification feature. {spec_doc_url}#Retry-Definition[Retries] are the only feature that Kogito does not support yet.
```

Comment:
It says that Retries are the only feature that Kogito does not support yet. However, workflow timeouts share the same emoji. How is it? Should we then change the emoji of workflow timeout to some experimental one? I believe it theoretically works with job service, right?

Reply:
Timeouts are not implemented yet, that's why it shares the same emoji. I'll change the text.